includes/MediaWiki/Api/EditEndpoint.php

```diff
@@ -203,9 +211,9 @@ public function needsWriteAccess() {
 	 */
 	public function getBodyValidator( $contentType ) {
 		if ( $contentType !== 'application/json' ) {
-			throw new HttpException( "Unsupported Content-Type",
-				415,
-				[ 'content_type' => $contentType ]
+			throw new LocalizedHttpException(
+				MessageValue::new( 'wikibasereconcileedit-editendpoint-invalid-reconcile-json' ),
```

Comment:
This also doesn’t seem like the right use case for that message?

Reply:
done